.ci/Jenkinsfile

```diff
@@ -179,6 +197,75 @@ pipeline {
         }
       }
     }
+    stage('Release') {
+      options {
+        skipDefaultCheckout()
+      }
+      when {
+        beforeAgent true
+        allOf {
+          branch 'master'
+          expression { return params.release }
+        }
+      }
+      stages {
+        stage('Notify') {
+          options { skipDefaultCheckout() }
+          steps {
+            notifyStatus(slackStatus: 'warning', subject: "[${env.REPO}] Release ready to be pushed",
+                         body: "Please go to (<${env.BUILD_URL}input|here>) to approve or reject within 12 hours.")
+          }
+        }
+        stage('Release CI') {
+          options { skipDefaultCheckout() }
+          input {
+            message 'Should we release a new version?'
+            ok 'Yes, we should.'
+          }
+          environment {
+            NPM_PACKAGE = '@elastic/synthetics'
+          }
+          steps {
+            cleanup()
+            withNodeEnv(){
+              dir("${BASE_DIR}"){
+                withNpmrc(secret: "${env.NPMRC_SECRET}", path: "${env.WORKSPACE}/${env.BASE_DIR}") {
+                  withGitRelease(credentialsId: '2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken') {
+                    sh(script: 'npm ci', label: 'prepare the ci environment')
+                    sh(script: 'npm version prerelease --preid=alpha', label: 'bump the alpha version')
+                    gitPush()
+                    gitPush(args: '--tags')
+                    setEnvVar('VERSION', sh(script: "npm view ${env.NPM_PACKAGE} version", returnStdout: true).trim())
+                    log(level: 'INFO', text: "Version to be published '${env.VERSION}'")
+                    withTotpVault(secret: "${env.TOTP_SECRET}", code_var_name: 'TOTP_CODE'){
+                      sh(script: "npm publish --otp=${env.TOTP_CODE} --tag alpha", label: 'publish to NPM')
```

Comment:
this should already invoke the `npm prepublish` which would clean our files. But just to be extra careful. can you add `npm run clean` during the release step to do a clean release.

Reply:
There is already a cleanup step earlier -> https://github.com/elastic/synthetics/pull/213/files#diff-ef8b2f5e4d21d81c0eba4754dea3a3dfc2c67d26284f195165b650cf4aff7beaR229

But let me add it for the shake of mind

Reply:
https://github.com/elastic/synthetics/pull/213/files#diff-ef8b2f5e4d21d81c0eba4754dea3a3dfc2c67d26284f195165b650cf4aff7beaR234 is now the clean step pre-release